### ISSUE TITLE
GTK Webkit?

### ISSUE THREAD

**Issue by codic12** [StefanSalewski/gintro#186]
Is it planned to add support for webkit? It could be useful in building small Nim gtk-based browsers.

**Comment by gavr123456789** [comment]
WebKit is already there

**Comment by codic12** [comment]
ah

**Comment by StefanSalewski** [comment]
>Is it planned to add support for webkit? It could be useful in building small Nim gtk-based browsers.

We added it about one year ago, as one of the authors of the famous Bacon/Dingle paper (inspiration of Nim ARC memory management) asked for it. But unfortunately that person vanished fast, he wants to continue using Python.

So we may remove webkit again? Indeed I am not able to test it for GTK4 currently, as my Gentoo Linux does not support webkit with GTK4. I was able to compile GTK4 with webkit support from git sources maybe 6 months ago, and have tested gintro with that successfully that time, but my last attempt to install from git failed.

Currently we have some trouble with libsoup 3.0, which is related to webkit.

Webkit is really a large module, which needs some other modules again. So when nobody uses it, maybe we should just remove it.
